sale_isolated_quotation/models/sale.py

```diff
@@ -0,0 +1,73 @@
+# © 2017 Ecosoft (ecosoft.co.th).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    order_sequence = fields.Boolean(readonly=True, index=True)
+    quote_id = fields.Many2one(
+        comodel_name="sale.order",
+        string="Quotation",
+        readonly=True,
+        ondelete="restrict",
+        copy=False,
+        help="For Sales Order, this field references to its Quotation",
+    )
+    order_id = fields.Many2one(
+        comodel_name="sale.order",
+        string="Order",
+        readonly=True,
+        ondelete="restrict",
+        copy=False,
+        help="For Quotation, this field references to its Sales Order",
+    )
+    quotation_state = fields.Selection(
+        string="Quotation Status",
+        readonly=True,
```

Comment:
no need this one while `related` field is default readonly.

Reply:
already fixed